spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/MappingSpannerReadConverter.java

```diff
@@ -115,44 +66,61 @@
 	public <R> R read(Class<R> type, Struct source, Set<String> includeColumns,
```

Comment:
Now that we've gone down this road of separating concerns and such, I don't think a method named "read" that reads from a source and gets names of pieces of data (columns) to read belongs in a "converter". 

I think we should have a separate SpannerEntityReader class and that this converter should not implement it.